
2021-12-05 10:01:19.920679: I tensorflow/stream_executor/cuda/cuda_dnn.cc:366] Loaded cuDNN version 8100
2021-12-05 10:01:20.661086: I tensorflow/stream_executor/cuda/cuda_blas.cc:1774] TensorFloat-32 will be used for the matrix multiplication. This will only be logged once.
Loaded model with validation_loss = 778.1830444335938
Epoch 1/500:
































































































































































































































































































































































1633/1633 [==============================] - 708s 433ms/step - train_loss: 878.8047
























270/279 [===========================>..] - ETA: 1s - valid_loss: 841.4855
279/279 [==============================] - 52s 184ms/step - valid_loss: 841.2263 - train_loss: 878.8047 - val_loss: 841.2263






































































































































































































































































































































































1633/1633 [==============================] - 718s 439ms/step - train_loss: 874.7465

























276/279 [============================>.] - ETA: 0s - valid_loss: 843.6631
279/279 [==============================] - 52s 185ms/step - valid_loss: 843.8381 - train_loss: 874.7466 - val_loss: 843.8381


























































































































































































































































































































































































































































































































































































1633/1633 [==============================] - 1145s 701ms/step - train_loss: 872.4736

























279/279 [==============================] - 53s 189ms/step - valid_loss: 824.7855 - train_loss: 872.4736 - val_loss: 824.7855
   0/1633 [..............................] - ETA: 0s - train_loss: 796.9674


































































































































































































































































































































































































































1633/1633 [==============================] - 838s 513ms/step - train_loss: 869.3679

























277/279 [============================>.] - ETA: 0s - valid_loss: 834.0532
279/279 [==============================] - 52s 185ms/step - valid_loss: 834.2363 - train_loss: 869.3678 - val_loss: 834.2363






































































































































































































































































































































































1633/1633 [==============================] - 720s 441ms/step - train_loss: 868.2068

























279/279 [==============================] - 52s 185ms/step - valid_loss: 848.5897 - train_loss: 868.2068 - val_loss: 848.5897
   1/1633 [..............................] - ETA: 24:02 - train_loss: 855.1021






































































































































































































































































































































































1633/1633 [==============================] - 720s 441ms/step - train_loss: 864.2233

























271/279 [============================>.] - ETA: 1s - valid_loss: 831.6754
279/279 [==============================] - 52s 185ms/step - valid_loss: 831.5363 - train_loss: 864.2234 - val_loss: 831.5363







































































































































































































































































































































































1633/1633 [==============================] - 721s 441ms/step - train_loss: 861.1683

























274/279 [============================>.] - ETA: 1s - valid_loss: 844.4133
279/279 [==============================] - 52s 185ms/step - valid_loss: 844.5101 - train_loss: 861.1683 - val_loss: 844.5101






































































































































































































































































































































































1633/1633 [==============================] - 721s 441ms/step - train_loss: 858.6458


























279/279 [==============================] - 52s 185ms/step - valid_loss: 845.4274 - train_loss: 858.6458 - val_loss: 845.4274
Epoch 9/500:



















































































































































































































































































1251/1633 [=====================>........] - ETA: 2:48 - train_loss: 857.2314
Traceback (most recent call last):
  File "train.py", line 40, in <module>
    train(model, training_data_generator, validation_data_generator, 500,\
  File "/media/avidbeam/workspace/Abdelrahman_Workspace/Project_Codes/Synergy/utils/custom_fit.py", line 32, in train
    train_loss = train_batch(X_batch,\
  File "/media/avidbeam/workspace/Abdelrahman_Workspace/Project_Codes/Synergy/utils/custom_fit.py", line 66, in train_batch
    y_pred = model(X, training=True)
  File "/home/avidbeam/.virtualenvs/TensorFlow_2/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 64, in error_handler
    return fn(*args, **kwargs)
  File "/home/avidbeam/.virtualenvs/TensorFlow_2/lib/python3.8/site-packages/keras/engine/base_layer.py", line 1083, in __call__
    outputs = call_fn(inputs, *args, **kwargs)
  File "/home/avidbeam/.virtualenvs/TensorFlow_2/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 92, in error_handler
    return fn(*args, **kwargs)
  File "/media/avidbeam/workspace/Abdelrahman_Workspace/Project_Codes/Synergy/model/synergy.py", line 70, in call
    Lr = self.encoder(Lc, Z, alpha_exp, alpha_shp)
  File "/home/avidbeam/.virtualenvs/TensorFlow_2/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 64, in error_handler
    return fn(*args, **kwargs)
  File "/home/avidbeam/.virtualenvs/TensorFlow_2/lib/python3.8/site-packages/keras/engine/base_layer.py", line 1083, in __call__
    outputs = call_fn(inputs, *args, **kwargs)
  File "/home/avidbeam/.virtualenvs/TensorFlow_2/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 92, in error_handler
    return fn(*args, **kwargs)
  File "/media/avidbeam/workspace/Abdelrahman_Workspace/Project_Codes/Synergy/model/encoder.py", line 141, in call
    X = self.conv3(X)
  File "/home/avidbeam/.virtualenvs/TensorFlow_2/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 64, in error_handler
    return fn(*args, **kwargs)
  File "/home/avidbeam/.virtualenvs/TensorFlow_2/lib/python3.8/site-packages/keras/engine/base_layer.py", line 1083, in __call__
    outputs = call_fn(inputs, *args, **kwargs)
  File "/home/avidbeam/.virtualenvs/TensorFlow_2/lib/python3.8/site-packages/keras/utils/traceback_utils.py", line 92, in error_handler
    return fn(*args, **kwargs)
  File "/home/avidbeam/.virtualenvs/TensorFlow_2/lib/python3.8/site-packages/keras/layers/convolutional.py", line 264, in call
    outputs = tf.nn.bias_add(
  File "/home/avidbeam/.virtualenvs/TensorFlow_2/lib/python3.8/site-packages/tensorflow/python/util/traceback_utils.py", line 150, in error_handler
    return fn(*args, **kwargs)
  File "/home/avidbeam/.virtualenvs/TensorFlow_2/lib/python3.8/site-packages/tensorflow/python/util/dispatch.py", line 1096, in op_dispatch_handler
    return dispatch_target(*args, **kwargs)
  File "/home/avidbeam/.virtualenvs/TensorFlow_2/lib/python3.8/site-packages/tensorflow/python/ops/nn_ops.py", line 3543, in bias_add
    return gen_nn_ops.bias_add(
  File "/home/avidbeam/.virtualenvs/TensorFlow_2/lib/python3.8/site-packages/tensorflow/python/ops/gen_nn_ops.py", line 674, in bias_add
    _result = pywrap_tfe.TFE_Py_FastPathExecute(

1254/1633 [======================>.......] - ETA: 2:47 - train_loss: 857.1281